src/hal/src/adapter.rs

```diff
@@ -127,6 +129,54 @@ pub trait PhysicalDevice<B: Backend>: fmt::Debug + Any + Send + Sync {
     fn is_valid_cache(&self, _cache: &[u8]) -> bool {
         false
     }
+
+    /// Enumerate active displays [surface][display::Display] from display.
+    /// Please notice that, even if a system has displays attached, they could be not returned because they are managed by some other components.
+    /// This function only return the display that are available to be managed by the current application.
+    /// Since, generally, while compositor are running they take the control of every display connected, it could be better to run the application directly from the tty to avoid the return of an empty list.
+    /// # Arguments
+    ///
+    /// * `adapter` - the [adapter][adapter::Adapter] from which the displays will be enumerated.
+    unsafe fn enumerate_available_displays(
```

Comment:
Yes, probably it is no needed. My initial thought was to made more explicit that the function will return only monitors that are available to be controlled by Vulkan and not every monitor, but since i have added this as notice in the function documentation, it is no more needed. I will change that.

Reply:
Done!